chore: Adding .prettier.json, updating README and package.json to have a standart format code.

RafaDSan opened the pull request:
I added the .prettierrc.json file to the root, prettier dependencies to the package.json and some new lines to the README to teach the users how to configure the prettier.json.

#48  closes

RafaDSan committed:
org: Adding .prettierrc.json to the root directory

RafaDSan committed:
org: Updating README.md to teach how to use prettier

RafaDSan committed:
fix: Adjusting README nested list in the "Stardard Formating Code" topic.

RafaDSan committed:
doc: Adding deployed addresses to README.md file and correcting grammatical mistakes

0xneves commented:
Could you revoke the additions of the prettier from the package.json @RafaDSan ?

RafaDSan commented:
> Could you revoke the additions of the prettier from the package.json @RafaDSan ?

Done! I removed the commit where I added the prettier plugin to the package.json. @0xneves
